src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --button-color: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408vh;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+img {
+  width: 160px;
+  height: 134px;
+  top: 33px;
+  left: 20px;
+}
+
+.card__t {
+  width: 166px;
+  height: 36px;
+  margin-top: 40px;
+  margin-bottom: 4px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-weight: bold;
```

Comment:
The 'font-weight' property is set twice consecutively with different values ('500' and 'bold'). This will cause the first declaration to be overridden. You should remove one of the declarations.